src/client/scripts/cardModal.js

```diff
@@ -0,0 +1,72 @@
+"use strict";
+/* global Cards:false SnackbarMethods:false UserMethods:false */
+
+Template.cardModal.helpers({
+    cardModalInfo: function(_id) {
+        return Cards.findOne({"_id": _id});
+    }
+});
+
+Template.cardModal.events({
+    "click #submitCommentButton": function(eve){
+        eve.preventDefault();
+        var comment = eve.target.form[0].value;
+        var image = null;
+
+        if(!comment || comment.length <= 4)
+            SnackbarMethods.DisplayMessage("Enter a more valuable comment",3000);//eslint-disable-line
+        else{
+            var author;
+
+            if(!Meteor.user())
+                author = Session.get("author");
+            else{
+                author = Meteor.user().profile.name;
+                image = UserMethods.getUserImage(Meteor.user()._id);
+            }
+            var commentToAdd = new Comment().createdBy(author)
+              .withText(comment).createdAtTime(new Date()).withProfile(image);
+
+            Meteor.call("submitComment",this._id,commentToAdd);
+            eve.target.form[0].value = "";
+            $("ul.collapsible li").show();
+            $("i.fa-caret-right").addClass("fa-caret-down");
+            $("i.fa-caret-right").removeClass("fa-caret-right");
+        }
+    },
+
+    "submit #submitCommentButton": function(eve){
```

Comment:
this looks like the same as above, im pretty sure you can do "click submit #submitCommentButton" and it will add this for both events.  Only difference I see is lines 33 and 34 which seem to just be changing css, and I dont see any difference in what is is doing between to 2 events